install/helm/agones/templates/controller.yaml

```diff
@@ -64,11 +64,21 @@ spec:
         release: {{ .Release.Name }}
         heritage: {{ .Release.Service }}
     spec:
-      {{- if .Values.agones.controller.affinity }}
```

Comment:
Removing this conditional makes it only possible to set the two types of affinity that we pre-define here. I don't know if anyone is using/overriding this parameter, but if so this would be a breaking change for them where their values.yaml would not work with the next release. 

Instead, I would suggest leaving this in (so that its still possible to set the affinity manually) but removing the config from `values.yaml` (as you have done). If affinity is explicitly set, then we use that. If not, then we do what you are doing here and move the default affinity into the template. 

We could also add a check that you don't both set a manual affinity block _and_ the new `requireDedicatedNode` to true at the same time, since that would not be a valid configuration. 

WDYT?

Reply:
Agreed. Fixed in the new revision.

Reply:
This is great, thanks!